gapic-generator-cloud/templates/cloud/client/method/def/_lro.erb

```diff
@@ -46,13 +47,19 @@ def <%= method.name %> request = nil, options: nil, **request_fields
   end
 
   request ||= request_fields
+  # request = Google::Gax::Protobuf.coerce request, to: <%= method.request_type %>
   request = Google::Gax.to_proto request, <%= method.request_type %>
 
-  operation = Google::Gax::Operation.new(
-    <%= method.ivar %>.call(request, options),
-    @operations_client,
-    call_options: options
-  )
-  operation.on_done { |operation| yield operation } if block_given?
-  operation
+  # Converts hash and nil to an options object
+  options = Google::Gax::ApiCall::Options.new options.to_h if options.respond_to? :to_h
+  header_params = {} # { name: request.name }
+  request_params_header = header_params.map { |k, v| "#{k}=#{v}" }.join("&")
+  metadata = @metadata.merge "x-goog-request-params".freeze => request_params_header
+  retry_policy = { retry_codes: [GRPC::Core::StatusCodes::UNAVAILABLE] }
```

Comment:
This may be okay for LRO, but I'm not sure so let's keep it consistent and remove it for now.

Reply:
Removed.